src/aspire/basis/fb.py

```diff
@@ -0,0 +1,55 @@
+import logging
+
+import numpy as np
+
+from aspire.basis.basis_utils import num_besselj_zeros
+from aspire.utils.matlab_compat import m_reshape
+
+logger = logging.getLogger(__name__)
+
+
+class FBBasisMixin(object):
+    """
+    FBBasisMixin is a mixin implementing methods specific to Fourier-Bessel expansions,
+    to be inherited by Fourier-Bessel subclasses of Basis.
+    """
+
+    def _getfbzeros(self):
```

Comment:
So I think we discussed renaming this according to the convention, something like `_calc_k_max` would be more appropriate.